src/npc.cpp

```diff
@@ -2293,5 +2281,17 @@ void npc::add_msg_player_or_npc(game_message_type type, const char *, const char
     va_end(ap);
 }
 
+void npc::add_new_mission( class mission *miss )
```

Comment:
Long-term fix for the 'kill something' mission is to convert them to
'return with a token of your kill', if a player goes out and butchers an
already-dead target, well how's the NPC know the difference?